bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/interaction/ServerHandles.java

```diff
@@ -21,17 +21,20 @@
 import org.eclipse.passage.lic.internal.jetty.JettyException;
 import org.eclipse.passage.lic.internal.jetty.JettyServer;
 import org.eclipse.passage.lic.internal.net.connect.Port;
+import org.eclipse.passage.lic.internal.net.connect.ServerBindAddress;
 
 final class ServerHandles extends Command {
 
 	private final Logger log = LogManager.getLogger(getClass());
 	private final JettyServer server;
-	private final Port port;
 	private final LicenseProtection license = new LicenseProtection();
+	private final ServerBindAddress serverBindAddress;
```

Comment:
compound name again

Reply:
please pay attention to [my comment above](https://github.com/eclipse-passage/passage/pull/1102#discussion_r909355006)